subo/command/deploy.go

```diff
@@ -0,0 +1,70 @@
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/suborbital/subo/deployer"
+	"github.com/suborbital/subo/project"
+	"github.com/suborbital/subo/subo/util"
+)
+
+var validDeployTypes = map[string]bool{
+	"kubernetes": true,
+	"k8s":        true,
+}
+
+//DeployCmd deploys the current project.
+func DeployCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deploy",
+		Short: "deploy an application",
+		Long:  "deploy the current project to a remote environment (Kubernetes, etc.)",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			deployType := args[0]
+			if _, valid := validDeployTypes[deployType]; !valid {
+				return fmt.Errorf("invalid deployment type %s", deployType)
+			}
+
+			cwd, err := os.Getwd()
+			if err != nil {
+				cwd = "$HOME"
```

Comment:
per @jagger27 , this can be replaced with `os.UserHomeDir()`

Reply:
There were actually a few places this was happening, an error should be returned instead of trying to default to $HOME in the first place. I've fixed them.